vars/main.yml

```diff
@@ -1,2 +1,4 @@
 ---
-# vars file for ansible-role-qemu-ceph-clients
\ No newline at end of file
+# vars file for ansible-role-qemu-ceph-clients
+script_debug: false
+exclude_no_rbd: false
```

Comment:
vars/ directory has quite much [1] higher priority than the defaults/ directory

To increase usability of the role they could be put in defaults/

script_debug is a quite generic name but I can't find it re-used in any other roles so probably fine.

[1]: https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable

Reply:
An unfortunate case of "I don't frequent this bar that often." Moved them to defaults, also renamed.

Reply:
I can't run with only "-e script_debug_ouput=True", have to also use the -e debug. 
Perhaps that's a feature?
Setting DEBUG=0 when script_debug_output=True caught me off card (and as a side bonus thing caused me to read the script a bit in more detail). If it's just me then keep as is, but somehow feels better that DEBUG=1 and EXCLUDE_=1 when debug is enabled and when we are excluding.

Reply:
Looks like the "empty" option in the ternary doesn't work as it sleaves a trail of double quotes

```
# ./ceph-version-qemu.sh "-d"
UUID
DEBUG: UUID has this many RBD devices configured: 0
```

works

```
# ./ceph-version-qemu.sh "" "-d"
UUID
UUID
```

doesn't.

I am starting to strongly dislike this task. :D

But stay tuned for a fix.

Reply:
Fixed